AirCasting/CreateSessionViews/SelectPeripheralView.swift

```diff
@@ -14,9 +14,11 @@ struct SelectPeripheralView: View {
     @EnvironmentObject var bluetoothManager: BluetoothManager
     @EnvironmentObject var sessionContext: CreateSessionContext
     @EnvironmentObject var connectionController: DefaultAirBeamConnectionController
+    @EnvironmentObject var sdSyncController: SDSyncController
     @Binding var creatingSessionFlowContinues: Bool
     let urlProvider: BaseURLProvider
     @EnvironmentObject var userAuthenticationSession: UserAuthenticationSession
+    var syncMode: Bool? = false
```

Comment:
I don't like this in-view branching based on domain-level bool flags. Let's leave it since I don't have a better solution just yet, but some day we'll need to revisit our UI architecture cuz it's pretty messy